main/ertp/guide/amount-math.md

```diff
@@ -11,35 +11,18 @@ The `AmountMath` library functions work for both fungible and nonfungible tokens
 There are two `AssetKinds`, each of which implements the same methods. Which kind is used 
```

Comment:
I remember you already have good text elsewhere that explains the current four asset kinds: nat, set, copySet, and copyBag. Could you correct it here as well, perhaps citing your existing text?